core/debug/src/core_armv7m/mpu_kinetis/debug_kinetis.c

```diff
@@ -47,20 +47,20 @@ static void debug_fault_mpu(void)
                     {
                         if(!found)
                         {
-                            dprintf("\n\r");
+                            default_putc('\n');
                             found = 1;
                         }
                         dprintf("    R%02d:", 11 - r);
                         for (i = 0; i < 4; i++) {
                             dprintf(" 0x%08X", MPU->WORD[11 - r][i]);
                         }
-                        dprintf("\n\r");
+                        default_putc('\n');
                     }
                 }
                 if(!found)
-                    dprintf("[none]\n\r");
-                dprintf("  Master port:      %d\n\r", (edr >> 4) & 0xF);
-                dprintf("  Error attribute:  %s %s (%s mode)\n\r",
+                    dprintf("[none]\n");
```

Comment:
since we are modifying this line - can you consider dding brackets around it.

Reply:
Fixed style in this file. It had a lot of style issues. It is very old code!